### PR TITLE
Craftable Lanyards

### PR DESCRIPTION
**Pull request by DukePaulAtreid3s**
## About the PR 
Adds recipes for lanyards at ID Fabricators, Uniform Fabricators, and the HoPs combo one.

## Why's this needed? 
They're useful little buggers and had  no buisness being as weirdly rare as they are.

## Changelog 

```changelog
(u)DukePaulAtreid3s
(*)Added a crafting recipe for lanyards-you can print them anywhere that prints IDs or uniforms.
```

### CODE REVIEW

File: code/datums/manufacturing.dm [hunk 1]
```diff
@@ -3186,6 +3186,13 @@ ABSTRACT_TYPE(/datum/manufacture/pod/weapon)
 
 
 /******************** HOP *******************/
+/datum/manufacture/lanyard
+	name = "Lanyard"
+	item_requirements = list("cloth" = 1)
+	item_outputs = list(obj/item/clothing/lanyard)
```
Comment (pgmzeta) [inline code/datums/manufacturing.dm:3192]: need starting slash
```suggestion
	item_outputs = list(/obj/item/clothing/lanyard)
```

File: code/datums/manufacturing.dm [hunk 2]
```diff
@@ -3186,6 +3186,13 @@ ABSTRACT_TYPE(/datum/manufacture/pod/weapon)
 
 
 /******************** HOP *******************/
+/datum/manufacture/lanyard
+	name = "Lanyard"
+	item_requirements = list("cloth" = 1)
```
Comment (pgmzeta) [inline code/datums/manufacturing.dm:3191]: item requirements use general classes, not specific materials
```suggestion
	item_requirements = list("fabric" = 1)
```

**Comment by Tarmunora** [comment]
Lanyards are intentionally primarily a mining lootcrate item, as part of the "useful" lootpool they pull from, and shouldn't be trivially printable at fabricators

**Comment by DukePaulAtreid3s** [comment]
> Lanyards are intentionally primarily a mining lootcrate item, as part of the "useful" lootpool they pull from, and shouldn't be trivially printable at fabricators

Are you saying to close this PR? Or to adjust it so they aren't trivially producible (I.E. through increasing the material cost and/or hacking the machine).

**Pull request closed without merging**